lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+
+
+
+ def dishes
+   {  :Burger => 5.00,
+      :Fries => 2.00,
+      :Pizza => 8.00,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)